dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -19,4 +19,4 @@
     appOptions.locale = '#{js_locale}';
     //]]>
 
-  %script{src: '/shared/js/shared.js'}
+  %script{src: "/shared/js/shared#{'.min' unless Rails.configuration.pretty_sharedjs}.js"}
```

Comment:
Is there a cleaner way to accomplish this? One thought I had was to make some sort of method like shared_path(path) that adds .min if necessary. Not sure where the best place for that to live would be, or if there's a yet better approach.